routes/index.js

```diff
@@ -1,9 +1,31 @@
 const apiRouter = require('express').Router();
+const { getAllProducts, getProductById } = require('../db/products');
 
-apiRouter.get("/", (req, res, next) => {
-  res.send({
-    message: "API is under construction!"
-  });
+apiRouter.get('/', (req, res, next) => {
+	res.send({
+		message: 'API is under construction!'
+	});
+});
+
+apiRouter.get('/products', async (req, res, next) => {
+	try {
+		const products = await getAllProducts();
+		console.log('products:', products);
+		res.send({
+			message: products
+		});
+	} catch (error) {}
+});
+
+apiRouter.get('/product/:productId', async (req, res, next) => {
```

Comment:
This route is not working right now.  Assumedly because the product is a string.  Casting it to a screen per previous comment should take care of this.

Reply:
Please please fix this route!